domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -23,6 +26,13 @@ public CQLWrapper() {
     super();
   }
 
+  public CQLWrapper(Criterion criterion) {
+    super();
```

Comment:
Can you remove this redundant `super()` in this an all other CQLWrapper constructors?

Reply:
Sure